src/implementations/twig/components/modal/modal.html.twig

```diff
@@ -0,0 +1,114 @@
+        },
+        ...
+      ],
+  Blocks:
+    - "header": free block to put any content in the modal header
+    - "body": free block to put any content in the modal body
+#}
+
+{# Internal properties #}
+
+{% set _id = id|default('') %}
+{% set _css_class = 'ecl-modal' %}
+{% set _icon_path = icon_path|default('') %}
+{% set _header_icon = header_icon|default({}) %}
+{% set _close_label = close_label|default('') %}
+{% set _header = header|default('') %}
+{% set _body = body|default('') %}
+{% set _buttons = buttons|default({}) %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% if attr.value is defined %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name|e('html_attr') ~ '="' ~ attr.value|e('html_attr') ~ '"' %}
+    {% else %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name|e('html_attr') %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<dialog id="{{ _id }}" class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  <div class="ecl-modal__container ecl-container">
+    <div class="ecl-modal__content ecl-col-12 ecl-col-m-8 ecl-offset-m-2 ecl-col-l-6 ecl-offset-l-3">
+      <header class="ecl-modal__header">
+        {% if _header_icon is not empty and _header_icon.icon is not empty %}
+          {% set _icon_extra_classes = 'ecl-modal__header-icon' %}
+          {% if _header_icon.extra_classes is not empty %}
+            {% set _icon_extra_classes = _icon_extra_classes ~ ' ' ~ _header_icon.extra_classes %}
+          {% endif %}
+          {% include '@ecl/icon/icon.html.twig' with _header_icon|merge({
+            icon: {
+              path: _header_icon.icon.path|default(_icon_path),
+              size: 'm',
+            }|merge(_header_icon.icon),
+            extra_classes: _icon_extra_classes,
+          }) only %}
+        {% endif %}
+
+        {% if _header is not empty -%}
+          <div class="ecl-modal__header-content">
+            {%- block header _header -%}
+          </div>
+        {% endif %}
+
+        {% include '@ecl/button/button.html.twig' with {
+          type: 'button',
+          variant: 'ghost',
+          label: _close_label,
+          icon: {
+            path: _icon_path,
+            name: 'close-filled',
+            size: 's',
+          },
+          hide_label: true,
+          extra_classes: 'ecl-modal__close',
+          extra_attributes: [{
+            name: 'data-ecl-modal-close',
+          }],
+        } only -%}
+      </header>
+      {% if _body is not empty %}
+      <div class="ecl-modal__body">{%- block body _body -%}</div>
+      {% endif %}
+      {% if _buttons is not empty %}
+      <footer class="ecl-modal__footer">
```

Comment:
that's what I did initially, a siple block in the footer. But according to the designer, the footer is meant to contain only buttons.
I guess that I could keep an empty block, just in case, and have the buttons parameter available too, to have them quickly